scipy/integrate/_ivp/bdf.py

```diff
@@ -422,6 +422,8 @@ def _step_impl(self):
             error_p_norm = np.inf
 
         error_norms = np.array([error_m_norm, error_norm, error_p_norm])
+        np.clip(error_norms, a_min=np.finfo(float).eps, a_max=None,
```

Comment:
OK, I just checked what happens here. I think it's fine to allow `factrors` to become `np.inf` when compute `0 ** -0.5` (for example). I suggest wrapping the power into
```Python
with np.errstate(divide='ignore'): 
   factors = error_norms ** (-1 / np.arange(order, order + 3))
```